define/dataDefinitions.go

```diff
@@ -0,0 +1,93 @@
+package define
```

Comment:
just struct declarations for the kind of objects used in the project

Reply:
The struct declarations in jsonhelper are precisely mirroring the json schema and are used for decoding, encoding
The structs in this file are designed more for usability.